mysite/javascript/googleanalytics.js

```diff
@@ -0,0 +1,17 @@
+(function(i,s,o,g,r,a,m){ i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-84547-17', 'auto', { 'allowLinker': true });
```

Comment:
This is the demo site so we wont want this changed. It was previously added via Page.php requirements so this is a step towards actually treating it like javascript ;)